prov/psm2/src/makefile

```diff
@@ -0,0 +1,7 @@
+setup:
+	cp ../../psm/src/psmx*.[ch] .
```

Comment:
Ah, I missed that.

How about having a `psm-rename.h` file that looks something like:

```c
#ifndef PSM_RENAME_H
#define PSM_RENAME_H

#ifdef PSMX_CONVERT_TO_PSMX2
#define psmx_foo psmx2_foo
#define psmx_bar psmx2_bar
// ...etc.
#endif

#endif
```

We do similar things in the OMPI code base for cases analogous to this.